### PR TITLE
update conductor helm chart, fix resource group names

### PR DESCRIPTION
**Pull request by nhudson**
* Update conductor helm chart
  * Support external secrets from Azure Key Vault
  * Support adding `podLabels` so that the pod will use Azure Workload Identity
  
* Update conductor code to rename the resource groups where the infrastructure lives.

### CODE REVIEW

File: charts/conductor/values.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ externalSecrets:
   parameterStore:
     name: "secret-store-parameter-store"
     kind: ClusterSecretStore
+  # remoteRef:
+  #   key: ~
+  #   secretKey: ~
```
Comment (ChuckHend) [inline charts/conductor/values.yaml:17]: not needed anymore or was this for debug?

Comment (nhudson) [reply charts/conductor/values.yaml:17]: Its commented out for reference.  We accept the values if they exist and don't if they don't.

Comment (nhudson) [reply charts/conductor/values.yaml:17]: I will add a comment explaining.

**Review by ChuckHend — APPROVED** [review]

File: charts/conductor/values.yaml (hunk 1 above)
Comment (ChuckHend) [reply charts/conductor/values.yaml:17]: cool just checking!

**Pull request merged**